### PR TITLE
Add questions tag

### PR DESCRIPTION
**Pull request by meatballs**
Because I find myself pointing people to [Asking Good Questions](https://www.pythondiscord.com/pages/guides/pydis-guides/asking-good-questions/) regularly.

### CODE REVIEW

**Review by CaedenPH — COMMENTED** [review]
Just my opinion, but this seems a little bare. Perhaps some other information could be useful

**Comment by meatballs** [comment]
What do you think it needs? The page it links to has comprehensive detail and I didn't want to just repeat any of that.

**Review by brodycritchlow — APPROVED** [review]
Don't see any point in adding anymore. The link provides enough detail. Looks good

**Review by Robin5605 — APPROVED** [review]
Short, concise, and links to a more detailed article. A perfect combination for tags. LGTM :+1:

File: bot/resources/tags/questions.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+**Asking Good Questions**
```
Comment (shtlrs) [inline bot/resources/tags/questions.md:1]: This needs to change to comply with the news tags slash command
#2419 is your friend

**Review by shtlrs — CHANGES_REQUESTED** [review]

**Comment by HassanAbouelela** [comment]
Blocked for further discussion, please open an issue in python-discord/meta.

**Comment by mbaruh** [comment]
Staff consensus in today's meeting is that we don't want to go forward with it, largely for the same reasons we removed the old !ask tag. The link doesn't clarify what exactly is wrong with the question asked, which is very context specific, and has a high chance of being used passive-aggressively by being dumped at people (sometimes several times) with no further explanation, as we have seen with the old tag.

**Pull request closed without merging**